worlds/osrs/__init__.py

```diff
@@ -0,0 +1,616 @@
+from BaseClasses import Item, Tutorial, ItemClassification, Region, Entrance, CollectionState, MultiWorld
+from worlds.AutoWorld import WebWorld, World
+from worlds.generic.Rules import add_rule
+from .Items import OSRSItem, starting_area_dict, chunksanity_starting_chunks, QP_Items
+from .Locations import OSRSLocation
+
+from .Options import OSRSOptions, StartingArea
+from .Names import LocationNames, ItemNames, RegionNames
+from .LogicCSVParser import load_location_csv, load_region_csv, load_resource_csv, load_item_csv
+
+
+class OSRSWeb(WebWorld):
+    theme = "stone"
+
+    setup_en = Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Oldschool Runescape Randomizer connected to an Archipelago Multiworld",
+        "English",
+        "docs/setup_en.md",
+        "setup/en",
+        ["digiholic"]
+    )
+    tutorials = [setup_en]
+
+
+class OSRSWorld(World):
+    game = "Old School Runescape"
+    option_definitions = OSRSOptions
```

Comment:
option_definitions is deprecated. Please use [options_dataclass.](https://github.com/NewSoupVi/Archipelago/blob/patch-46/docs/options%20api.md#option-creation)

(The text is a bit outdated but the code examples are correct)